File: Soal 1.cpp
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;

int main() {
    // Accessing elements
    vector<int> arr = {1, 2, 3, 4, 5, 6};
    cout << arr[5] << endl;  

    // Updating elements
    arr[2] = 17;
    for (int i : arr) {
        cout << i << " ";
    }
    cout << endl;  

    // Inserting elements
    arr.insert(arr.begin() + 2, 25);
    for (int i : arr) {
        cout << i << " ";
    }
    cout << std::endl;  

    // Deleting elements
    arr.erase(arr.begin() + 5);
    for (int i : arr) {
        cout << i << " ";
    }
    cout << endl;  
    // Searching elements
    bool found = find(arr.begin(), arr.end(), 25) != arr.end();
    cout << (found ? "True" : "False") << endl;  
    // Traversal
    for (int i : arr) {
        cout << i << " ";
    }
    std::cout << endl;

    // Sorting
    sort(arr.begin(), arr.end());
    for (int i : arr) {
        cout << i << " ";
    }
   cout << endl;  
    // Merging
    vector<int> arr2 = {9, 10, 11};
    arr.insert(arr.end(), arr2.begin(), arr2.end());
    for (int i : arr) {
        cout << i << " ";
    }
    cout << endl;  
    return 0;
}

File: Soal 1.py
# Akses Elemen
arr = [1, 2, 3, 4, 5, 6 ]
print(arr[2])  

# Pembaruan Elemen
arr[5] = 17
print(arr)  

# Insertion
arr.insert(1, 50)
print(arr) 

# Deletion
del arr[4]
print(arr)  

# Pencarian Elemen
print(20 in arr)  

# Traversal
for i in arr:
    print(i)

#Sorting
arr.sort()
print(arr)  

# Merging
arr2 = [9, 10, 11]
arr.extend(arr2)
print(arr)  